jest.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  transform: {
+    '^.+\\.jsx?$': '<rootDir>/jest-preprocess.js',
+    '^.+\\.tsx?$': 'ts-jest',
+  },
+  testRegex: '\\.spec\\.([tj]sx?)$',
```

Comment:
Files with `/*.spec.tsx` extension are run with jest.

Reply:
(not a veto) In the long run I think we ought to use only one testing framework (so just jest)